### ISSUE TITLE
How to set z-index for ngx-ui-loader

### ISSUE THREAD

**Issue by varund29** [t-ho/ngx-ui-loader#72]
**Describe the bug**

**To Reproduce**
**Please provide basic setup/reproduction on [Stackblitz](https://stackblitz.com/)**

**Expected behaviour**

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

**Comment by ujjaldey** [comment]
True. There should be an option to configure the z-index. I am using confirmation modal in canDeactivate. But the modal is not clickable due to the overlay.

**Comment by jhormanrus** [comment]
+1
